src/components/WorkQueueCombobox.vue

```diff
@@ -49,20 +51,68 @@
   const api = useWorkspaceApi()
   const workQueuesSubscription = useSubscription(api.workQueues.getWorkQueues, [{}])
```

Comment:
@desertaxle - Collin is updating an older component to use with work queue related automations. If we continue to use the [workspace/get_work_queues](https://api.stg.prefect.dev/api/accounts/{account_id}/workspaces/{workspace_id}/work_queues/filter) route, will this conflict with future plans to remove api work queue routes after agents are deprecated?

Reply:
to be fair, these changes don't really lean _more_ into that route. Another way to look at this is I'm fixing a bug where these components are currently being used? 

For automations, we have another component that wraps this one which I'm going to be using so _transitively_ it's another use of this route but still bounded/contained in the same way as the status quo.

Reply:
Yep that's fair.  I'm just keen for us to make sure that we are coordinating a few different pieces of work.  Worst case scenario is that we have to update this component again in the future so my comment is non-blocking.

Reply:
👍 in slack